ci/Dockerfile.lambda

```diff
@@ -36,7 +37,7 @@ RUN chmod 755 /usr/bin/aws-lambda-rie /entry.sh /sync_lambda_envs.sh
 # New Relic lambda layer
 RUN unzip newrelic-layer.zip -d /opt && rm newrelic-layer.zip
 
-ENTRYPOINT [ "/entry.sh" ]
+ENTRYPOINT [ "sh", "-c", "poetry run /entry.sh" ]
```

Comment:
not sure if you need the "sh", "-c" here?

Reply:
I ran `docker run --entrypoint="poetry run /entry.sh" api-test-lambda` and ended up with 
```
unable to start container process: exec: "poetry run /entry.sh": stat poetry run /entry.sh: no such file or directory: unknown.
```

Reply:
ah, interesting!